Source/WebKit/Shared/Cocoa/CoreIPCDDScannerResult.serialization.in

```diff
@@ -20,12 +20,16 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#if PLATFORM(COCOA) && ENABLE(DATA_DETECTION)
-
-webkit_platform_headers: "CoreIPCDDScannerResult.h"
-
-[WebKitPlatform] class WebKit::CoreIPCDDScannerResult {
-    WebKit::CoreIPCDDScannerResult::Value m_value
+#if ENABLE(DATA_DETECTION)
+[WebKitSecureCodingClass=PAL::getDDScannerResultClass(), WebKitSecureCodingHeader=<pal/cocoa/DataDetectorsCoreSoftLink.h>] webkit_secure_coding DDScannerResult {
+    AR: NSValue
+    MS: String
+    T: String
+    P: Number
+    VN: Number
+    SR: ?Array<DDScannerResult>
+    V: ?String
```

Comment:
I think the syntax would be more conventional if the question mark were at the end.
Also, are these 1-2 letter strings really the strings to use?  I kind of like variable names that indicate what they are for.

Reply:
This are literally the keys put in the dictionary by the other framework, so yes they are correct. We have no flexibility on that.

I can move the ? to the end though